jsettlers.logic/src/jsettlers/logic/movable/MovableStrategy.java

```diff
@@ -45,10 +46,12 @@
 public abstract class MovableStrategy implements Serializable {
 	private static final long serialVersionUID = 3135655342562634378L;
 
-	private final Movable movable;
+	protected final Movable movable;
+	protected final AbstractMovableGrid grid;
```

Comment:
I don't see a big difference between getGrid().... and grid....
I think we should not add this field simply to make the code a few characters shorter.